include/zephyr/arch/riscv/exp.h

```diff
@@ -49,6 +53,9 @@ struct soc_esf {
 #endif
 #endif
 
+#if defined(CONFIG_RISCV_SOC_ISR_STACKING)
+	SOC_ISR_STACKING_ESF_DECLARE
```

Comment:
```suggestion
	SOC_ISR_STACKING_ESF_DECLARE;
```

Reply:
uhm, TBH I hate that :) If I'm declaring a struct somewhere I expect that to be declared already with an trailing semicolon.

Reply:
Afaik we tipically do that in other places, e.g. `DEVIC_DECLARE();`. It's something useful e.g. for formatters.

Reply:
Btw, https://docs.kernel.org/dev-tools/checkpatch.html -> see `TRAILING_SEMICOLON`

Reply:
changed